R/utils.R

```diff
@@ -29,3 +29,91 @@ resolve_conflicts <- function() {
   conflicted::conflict_prefer("summarise", "dplyr")
   conflicted::conflict_prefer("arrange", "dplyr")
 }
+
+#' @examples
+#' some_dockerfile <- tempfile()
+#' writeLines(create_empty_dockerfile("library(dplyr)"), some_dockerfile)
+#'
+#' # Before
+#' writeLines(readLines(some_dockerfile))
+#' update_dockerfile_packages(path = some_dockerfile)
+#' # After
+#' writeLines(readLines(some_dockerfile))
```

Comment:
Here is how this works:

``` r
  
devtools::load_all()
#> Loading PACTA.analysis
some_dockerfile <- tempfile()
writeLines(create_empty_dockerfile("library(dplyr)"), some_dockerfile)

# Before
writeLines(readLines(some_dockerfile))
#> # update-dockerfile-packages-start
#> library(dplyr)
#> # update-dockerfile-packages-end
update_dockerfile_packages(path = some_dockerfile)
# After
writeLines(readLines(some_dockerfile))
#> # update-dockerfile-packages-start
#>     && Rscript -e "install.packages( \
#>              c( \
#>                'assertthat', \
#>                'bookdown', \
#>                'config', \
#>                'conflicted', \
#>                'countrycode', \
#>                'data.table', \
#>                'devtools', \
#>                'dplyr', \
#>                'fs', \
#>                'fst', \
#>                'ggplot2', \
#>                'glue', \
#>                'here', \
#>                'janitor', \
#>                'jsonlite', \
#>                'knitr', \
#>                'purrr', \
#>                'readr', \
#>                'readxl', \
#>                'renv', \
#>                'reshape2', \
#>                'rlang', \
#>                'rmarkdown', \
#>                'rstudioapi', \
#>                'scales', \
#>                'stringr', \
#>                'testthat', \
#>                'tibble', \
#>                'tidyr', \
#>                'tidyselect', \
#>                'usethis', \
#>                'withr', \
#>                'writexl', \
#>                'zoo' \
#>              ) \
#>            )" \
#> # update-dockerfile-packages-end
```

<sup>Created on 2020-10-13 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>